js/src/scraper/scraper.link.js

```diff
@@ -0,0 +1,28 @@
+var scrapers = require( "./../scraper-store.js" );
+
+var Scraper = function() {};
+
+Scraper.prototype.scrape = function(fields){
+
+    var that = this;
+
+    fields = _.map(fields, function(field){
```

Comment:
You might consider returning this directly like `return _.map(fields, function(field){`